checks/apps/namd/namd_check.py

```diff
@@ -1,133 +1,296 @@
-# Copyright 2016-2022 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
 # ReFrame Project Developers. See the top-level LICENSE file for details.
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
 import os
+from types import FunctionType
```

Comment:
is FunctionType used ?

Reply:
No, well spotted. I was trying to use it, but then understood how to do things in a cleaner way.